includes/specials/SpecialUserlogout.php

```diff
@@ -44,8 +44,11 @@ function execute( $par ) {
 			throw new HttpError( 400, wfMessage( 'suspicious-userlogout' ), wfMessage( 'loginerror' ) );
 		}
 
-		// Clean up any facebook cookies/data
-		FacebookClient::getInstance()->logout();
+		// Some communities (notably uncyclopedia) do not have FacebookClient enabled
+		if ( F::app()->wg->EnableFacebookClientExt ) {
+			// Clean up any facebook cookies/data
+			FacebookClient::getInstance()->logout();
+		}
```

Comment:
Please use a [hook](https://www.mediawiki.org/wiki/Manual:Hooks/UserLogoutComplete) to avoid adding our code (wrapped in an if statement) into MW code.